trade-finance-manager-api/api-tests/v1/deals/deals-submit.second-submission.api-test.js

```diff
@@ -149,7 +150,7 @@ describe('/v1/deals', () => {
         });
 
         it('should add bond.facilityGuaranteeDates', async () => {
-          const initialBond = MOCK_DEAL_AIN_SECOND_SUBMIT_FACILITIES_UNISSUED_TO_ISSUED.bondTransactions.items[0];
+          const initialBond = cloneDeep(MOCK_DEAL_AIN_SECOND_SUBMIT_FACILITIES_UNISSUED_TO_ISSUED.bondTransactions.items[0]);
```

Comment:
Nice spot :)

Reply:
I didn't look for any more but suspicious these will be widespread...